File: 3227.py
class Solution:
    def doesAliceWin(self, s: str) -> bool:
        for c in s:
            if c in ['a', 'e', 'i', 'o', 'u']:
                return True
        return False
        
if __name__ == '__main__':
    s = Solution()
    print(s.doesAliceWin(input()))

File: 3227.cpp
#include<iostream>
#include<string>

using namespace std;

class Solution {
public:
    bool doesAliceWin(string s) {
        for(char c : s){
            if(c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u') return true;
        }
        return false;
    }
};

int main(){
    Solution s;
    string str;
    cin >> str;
    cout << s.doesAliceWin(str) << endl;
}